src/models/BaseRdfConcept.py

```diff
@@ -53,20 +53,22 @@ def get_gpp_link(  # noqa: C901 #TODO
         url_parts = urlparse("https://zoeken.beeldengeluid.nl/")
         parents = metadata.get("parents")
 
-        if cat_type.upper() == "SEASON" and parents not in (None, []):
+        if cat_type.upper() == "SEASON":
```

Comment:
It looks to me that this code tries to 'get' something from a list, which only works if 'it' is a dict.
```python
        # parents can be a dict, list or none. turn that into a (possibly empty) list.
        parents_is_dict = isinstance(metadata.get("parents"), dict)
        parents: list = (
            [metadata["parents"]] if parents_is_dict else metadata.get("parents", [])
        )
```
For seasons, we are trying to get the series id using get on parent, but parent can be list. 
```python
        elif cat_type_lower == "season":
            series_ids = [
                parent.get("parent_id")
                for parent in parents
                if parent.get("parent_type") == "SERIES"
            ]
```
So, parent.get will raise an exception:
`AttributeError: 'list' object has no attribute 'get'` when the parent is a list (not the empty one).

Reply:
One more difference to the original code: when a season has a list of parents, the first parent listed with was used to construct the series URN if it was of type SERIES .
```python
            if isinstance(parents, list):
                if parents[0].get("parent_type") == "SERIES":
                    parent_id = parents[0].get("parent_id")
                    series_urn = f"urn:vme:default:series:{parent_id}"
```
Now this may not be the best approach, because if the first parent isn't a SERIES, but the second is, then this is missed, but the new code dismisses the series URN completely if a season has multiple parents of type SERIES.
```python
            series_id = series_ids[0] if len(series_ids) == 1 else None
```
We may have to be more loose on this, otherwise we miss the url for this season completely. So I think this could be:
```python
            series_id = series_ids[0] if len(series_ids) >= 1 else None
```
So, we then just pick the first parent that is a series from the parents list.